admin-tools/src/webapp/vm/aliases/chef_aliases_list.vm

```diff
@@ -6,7 +6,7 @@
 	<h3>
 		$tlang.getString("alias.aliases")
 	</h3>
-	#if ($alertMessage)<div class="alertMessage">$tlang.getString("alias.list.alert") $validator.escapeHtml($alertMessage)</div>#end
+	#if ($alertMessage)<div class="sak-banner-error">$tlang.getString("alias.list.alert") $validator.escapeHtml($alertMessage)</div>#end
```

Comment:
Here is a third way of doing this...

I don't see an English translation for "alias.list.alert". Let's update these to all use the same key.